<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlignmentDialog</class>
 <widget class="QDialog" name="AlignmentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>1058</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alignment Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Peak Selection</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Minumum Peak Intensity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="minGroupIntensity">
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>5000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Minimum Peak S/N Ratio</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="minSN">
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Minimum Peak Width</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="minPeakWidth">
        <property name="suffix">
         <string> scans</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Peak Detection Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="peakDetectionAlgo">
        <item>
         <property name="text">
          <string>Compound Database Search</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Automated Peak Detection</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="selectDatabase">
        <property name="text">
         <string>Select Database</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="selectDatabaseComboBox"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Min Intensity for Scans</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="minIntensity">
        <property name="maximum">
         <double>9999999999.989999771118164</double>
        </property>
        <property name="value">
         <double>50000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Max Intensity for Scans</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="maxIntensity">
        <property name="maximum">
         <double>9999999999.989999771118164</double>
        </property>
        <property name="value">
         <double>9999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="2">
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="alignButton">
        <property name="text">
         <string>Align</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="statusText">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="UndoAlignment">
        <property name="text">
         <string>Undo Aligment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Group Selection Criteria for use in Alignment</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Group must contain at least  [X] good peaks</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="minGoodPeakCount">
        <property name="suffix">
         <string> peaks</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Limit total number of groups in alignment to</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="limitGroupCount">
        <property name="suffix">
         <string> groups</string>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Peak Grouping Window</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="groupingWindow">
        <property name="suffix">
         <string> scans</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="alignWrtExpectedRt">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Alignment w.r.t Expected Rt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Alignment Algorithm</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Alignment Algorithm</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="alignAlgo">
        <property name="currentText">
         <string>Poly Fit</string>
        </property>
        <item>
         <property name="text">
          <string>Poly Fit</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OBI-Warp [Beta]</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Loess Fit  [Deprecated]</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Maximum Number of Iterations</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="maxIterations">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Polynomial Degree</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="polynomialDegree">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelScoreObi">
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QComboBox" name="scoreObi">
        <item>
         <property name="text">
          <string>cor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>cov</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>prd</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>euc</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelResponseObiWarp">
        <property name="text">
         <string>Response</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="responseObiWarp">
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>20.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="labelBinSizeObiWarp">
        <property name="text">
         <string>Bin Size</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="binSizeObiWarp">
        <property name="maximum">
         <double>2.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.600000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelGapInit">
        <property name="text">
         <string>Gap-init</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="gapInit">
        <property name="maximum">
         <double>97.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>0.200000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="labelGapExtend">
        <property name="text">
         <string>Gap-extend</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QDoubleSpinBox" name="gapExtend">
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>3.400000000000000</double>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="labelFactorDiag">
        <property name="text">
         <string>Factor-diag</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QDoubleSpinBox" name="factorDiag">
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>2.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="labelFactorGap">
        <property name="text">
         <string>Factor-gap</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QDoubleSpinBox" name="factorGap">
        <property name="singleStep">
         <double>0.500000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="labelNoStdNormal">
        <property name="text">
         <string>No std normal</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QCheckBox" name="noStdNormal">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="labelLocal">
        <property name="text">
         <string>Local</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QCheckBox" name="local">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="labelInitPenalty">
        <property name="text">
         <string>Init. penalty</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QDoubleSpinBox" name="initPenalty"/>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="labelRestoreDefaultObiWarpParams">
        <property name="text">
         <string>Restore Default Values</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QPushButton" name="restoreDefaultObiWarpParams">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelShowAdvanceParams">
        <property name="text">
         <string>Show Advance Params</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="showAdvanceParams">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="refSampleLabel">
        <property name="text">
         <string>Choose reference sample</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="samplesBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minGoodPeakCount</tabstop>
  <tabstop>limitGroupCount</tabstop>
  <tabstop>groupingWindow</tabstop>
  <tabstop>minGroupIntensity</tabstop>
  <tabstop>minSN</tabstop>
  <tabstop>minPeakWidth</tabstop>
  <tabstop>peakDetectionAlgo</tabstop>
  <tabstop>selectDatabaseComboBox</tabstop>
  <tabstop>minIntensity</tabstop>
  <tabstop>maxIntensity</tabstop>
  <tabstop>alignAlgo</tabstop>
  <tabstop>maxIterations</tabstop>
  <tabstop>polynomialDegree</tabstop>
  <tabstop>UndoAlignment</tabstop>
  <tabstop>alignButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
